New reference pages from Postgres 8.3

janebeckman opened the pull request:
Pages for ALTER/CREATE/DROP OPERATOR FAMILY, DISCARD, and ALTER VIEW. New navigation links.

mkiyama commented:
I am kind of confused. Did you replace existing commands?
If you did, I think the new commands are new, additional commands, not replacement commands.

mkiyama commented:
Did you update this page?
http://gpdb.docs.pivotal.io/43120/ref_guide/feature_summary.html

There is a section on Greenplum and PostgreSQL compatibility

mkiyama commented:
SQL summary pages also need to be updated 

 gpdb/gpdb-doc/dita/ref_guide/GPReferenceSQLSummaryLOP.xml
 gpdb/gpdb-doc/dita/client_tool_guides/client/unix/client_sql_ref.xml

dyozie commented:
@mkiyama - I'm not sure why this PR is showing changed files instead of new files, but it does look like it adds the new files.  Also, just for clarity, I think you mean this should update the http://gpdb.docs.pivotal.io/500Alpha/ref_guide/feature_summary.html page (5.0 rather than 4.3.x).

mkiyama commented:
@dyozie 
The original pull request had the new SQL commands replacing related commands. Jane fixed that issue.
There are other topics that require updating.
-The command summary pages that I mentioned (in the ref guide and in client guide.)
-The feature summary page that lists the Postgres features supported by GPDB that you listed.

Does that answer your questions?

janebeckman commented:
Updated the remaining files with links. Thanks for the pointer to the additional summary pages. I also updated the summary page with CREATE/ALTER/DROP EXTENSION commands, but left those out of GPReferenceSQLSummaryLOP.xml and client_sql_ref.xml, since those used conrefs.

janebeckman commented:
Checking some comments from Heikki. Might be one more checkin on compatibility.

janebeckman commented:
Eliminated some restrictions and notes per Heikki.

dyozie commented on gpdb-doc/dita/ref_guide/feature_summary.xml:
```diff
@@ -425,6 +438,12 @@ NEXT 10 ROWS ONLY; </codeblock><p>Greenplum
             <row>
+              <entry colname="col1"><codeph>CREATE EXTENSION</codeph></entry>
+              <entry colname="col2">YES</entry>
+              <entry colname="col3">Loads a new extension into Greenplum Database -  not in
```
Instead of saying "not in PostgreSQL 8.3" let's just say what version it was based on.  Readers shouldn't expect behavior to be in line with 8.3.

dyozie commented on gpdb-doc/dita/ref_guide/feature_summary.xml:
```diff
@@ -39,9 +39,6 @@
           <li id="ik264434">Some set returning subqueries in <codeph>EXISTS</codeph> or <codeph>NOT
               EXISTS</codeph> clauses that Greenplum's parallel optimizer cannot rewrite into
             joins.</li>
-          <li id="ik264454"><codeph>UNION ALL</codeph> of joined tables with subqueries.</li>
```
Heikki mentioned that he didn't think there were any limitations with UNION ALL.  Why is it removed?

janebeckman replied:
It's removed because it's part of a list following the statement "The following SQL constructs are not supported:"
